### PR TITLE
Use the new oracle's expiration time instead of inserted+ttl

### PR DESCRIPTION
**Pull request by Arachnid**

### CODE REVIEW

**Comment by makoto** [comment]
@Arachnid The CI test is failing

File: src/oracle.ts [hunk 1]
```diff
@@ -17,6 +17,11 @@ export class OutdatedDataError extends Error {
     }
 }
 
+// Compares two serial numbers using RFC1982 serial number math.
+function serialNumberGt(i1: number, i2: number): boolean {
+    return (i1 < i2 && i2 - i1 > 2147483648) || (i1 > i2 && i1 - i2 < 2147483648);
```
Comment (makoto) [inline src/oracle.ts]: Please add a comment about 2147483648 being `2^31` and a link to https://tools.ietf.org/html/rfc1982

Comment (Arachnid) [reply src/oracle.ts]: I've changed it to use the hex constant to make it clearer.

**Comment by Arachnid** [comment]
CI is failing because I have not yet published the new DNSSEC oracle version this change depends on. PTAL!

See also https://github.com/ensdomains/dnsregistrar/pull/40 and https://github.com/ensdomains/dnssec-oracle/pull/162

**Review by decanus — APPROVED** [review]

File: src/oracle.ts (hunk 1 above)
Comment (makoto) [reply src/oracle.ts]: Did you push the change? I still see `return (i1 < i2 && i2 - i1 > 2147483648) || (i1 > i2 && i1 - i2 < 2147483648);`

**Comment by Arachnid** [comment]
PTAL

**Review by makoto — APPROVED** [review]

**Pull request merged**